### DISCUSSION TITLE
MTA-STS & TLSRPT support

### DISCUSSION THREAD

**Discussion by HLFH** [postalserver/postal#1485] [Feature suggestions]
Hi,

It would be great to support MTA-STS & TLSRPT into Postal.

https://tools.ietf.org/html/rfc8461

Thanks,
HLFH

**Comment by willpower232** [comment]
This sounds like a great standard but at the very least it will mean adding a lot of extra checks into the code of Postal and without widespread adoption, I'd reckon its not going to be a priority.

More reading:
- https://www.hardenize.com/blog/mta-sts
- https://security.googleblog.com/2019/04/gmail-making-email-more-secure-with-mta.html

**Reply by dragoangel** [reply]
Agree that this a hard work. Standard looks easy for implementing on server rcpt side, but server sndr side implementation are hell hard.
